### PR TITLE
Deprecate xdr-run-script command and related playbook

### PR DESCRIPTION
**Pull request by tcarmeli1**
## Status
- [x] Ready

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-11109

## Description
Deprecate xdr-run-script command and related playbook

### CODE REVIEW

File: Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.yml [hunk 1]
```diff
@@ -2252,8 +2252,9 @@ script:
     - defaultValue: '600'
       description: The timeout in seconds for this execution.
       name: timeout
-    description: This command will soon be deprecated; run xdr-script-run instead. Initiates a new endpoint script execution action using a script from the script library.
+    description: Deprecated. run xdr-script-run instead.
```
Comment (ShirleyDenkberg) [inline Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.yml:2255]:
```suggestion
    description: Deprecated. Run xdr-script-run instead.
```

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Run_script_README.md [hunk 2]
```diff
@@ -1,5 +1,4 @@
-Initiates a new endpoint script execution action using a provided script unique id from Cortex XDR script library.
-
+Deprecated. Use the `xdr-script-run` command instead. Initiates a new endpoint script execution action using a provided script unique id from Cortex XDR script library.
```
Comment (ShirleyDenkberg) [inline Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Run_script_README.md:1]:
```suggestion
Deprecated. Use the `xdr-script-run` command instead. Initiates a new endpoint script execution action using a provided script unique ID from Cortex XDR script library.
```

File: Packs/CortexXDR/ReleaseNotes/6_1_84.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+##### Palo Alto Networks Cortex XDR - Investigation and Response
+
+- Deprecated ***xdr-run-script*** command. Use ***xdr-script-run*** instead.
+
+#### Playbooks
+
+##### Cortex XDR - Run script
+
+- Deprecated. Use the `xdr-script-run` command instead.
```
Comment (ShirleyDenkberg) [inline Packs/CortexXDR/ReleaseNotes/6_1_84.md:12]:
```suggestion
Deprecated the ***xdr-run-script*** command. Use ***xdr-script-run*** instead.

#### Playbooks

##### Cortex XDR - Run script

Deprecated. Use the `xdr-script-run` command instead.
```

**Comment by ShirleyDenkberg** [comment]
@maimorag @jbabazadeh Doc review completed.

**Review by jbabazadeh — APPROVED** [review]
Great!!

**Pull request merged**